common/dynamicconfig/constants.go

```diff
@@ -374,6 +374,16 @@ If exceeded, failure will be truncated before being stored in mutable state.`,
 		16,
 		`MutableStateTombstoneCountLimit is the maximum number of deleted sub state machines tracked in mutable state.`,
 	)
+	MutableStateMaxPausedActivityTypeCount = NewNamespaceIntSetting(
+		"limit.mutableStateMaxPausedActivityTypeCount",
+		10,
+		`MutableStateMaxPausedActivityTypeCount is the maximum number of activity types that can be paused at once.`,
+	)
+	MutableStateMaxPausedActivityTypeLength = NewGlobalIntSetting(
```

Comment:
Do we have a config to limit the activity name length that I can reuse? I searched and couldn't find any.